.env

```diff
@@ -1,5 +1,6 @@
 SKIP_PREFLIGHT_CHECK=true
 ACCESS_API_URL=https://coleman-access-api.dev.insee.io
+MY_SURVEYS_URL=https://mes-enquetes.developpement.insee.fr
```

Comment:
This default value is not accesible outside Insee network.
Can you update this ?

Reply:
Value changed